exporter/datadogexporter/translate_traces.go

```diff
@@ -148,7 +150,18 @@ func resourceSpansToDatadogSpans(rs pdata.ResourceSpans, calculator *sublayerCal
 		extractInstrumentationLibraryTags(ils.InstrumentationLibrary(), datadogTags)
 		spans := ils.Spans()
 		for j := 0; j < spans.Len(); j++ {
-			span := spanToDatadogSpan(spans.At(j), resourceServiceName, datadogTags, cfg)
+			otelSpan := spans.At(j)
+
+			// TODO: until zipkin duplicate span ids are clarified, drop duplicate spans.
+			// Update this behavior when the behavior is finalized by spec
+			// https://github.com/open-telemetry/opentelemetry-specification/issues/1007
+			if duplicateSpanID := spanMap[otelSpan.SpanID().HexString()]; duplicateSpanID {
+				continue
+			} else {
+				spanMap[otelSpan.SpanID().HexString()] = true
+			}
```

Comment:
nit (since the `if` branch has a `continue`)
```suggestion
			spanMap[otelSpan.SpanID().HexString()] = true
```